src/constants/burgerSeeds.ts

```diff
@@ -14,6 +14,8 @@ export const SEED_PROGRAM_DELEGATE = Buffer.from(JSON.parse(
     })[0].value
 ));
 
+export const GAME_CONFIG_SEED = "GAME_CONFIG";
```

Comment:
This seed needs to be read like the other seeds and not hardcoded
ask bob when in doubt